blocks/header/header.css

```diff
@@ -312,6 +308,19 @@ header nav .nav-tools {
 
 
 @media (min-width: 900px) {
+  header nav .nav-brand:not(:has(img)) {
```

Comment:
`:has` is not supported on firefox yet https://caniuse.com/css-has

Reply:
I removed all `:has` except for this one. I couldn't think of a simple way of doing this. Firefox users will just have a small layout shift before the image is loaded